Configurations/unix-Makefile.tmpl

```diff
@@ -1583,6 +1583,16 @@ EOF
           my $section = $1;
           my $name = uc basename($args{src}, ".$section");
           my $pod = $gen0;
+
+          if ($config{manpage_format} eq "mdoc") {
+              return <<"EOF";
+$args{src}: $pod
+	pod2mdoc -n $name -s $section\$(MANSUFFIX) \\
+		 -d \$(RELEASE_DATE) \\
+		 $pod >\$\@
+EOF
+          }
+          die "Unhandled manpage format: $config{manpage_format}" unless ($config{manpage_format} eq "roff");
```

Comment:
Should the generation of the roff format below be skipped if the config value is mdoc?